### ISSUE TITLE
Create workspaces from OAI-PMH links

### ISSUE THREAD

**Issue by wrznr** [OCR-D/core#539]
Many libraries deliver METS files via a machine-readable OAI-PMH interface. E.g.,
![image](https://user-images.githubusercontent.com/26707219/87673582-12478700-c775-11ea-9ee5-fbd816b5ab36.png)

with https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-497166623

Such links are not compatible with the `ocrd workspace clone` command. Since the direct link to the METS file is not provided, it is hard to create a workspace in this context. It would be fantastic if a harvesting option for OAI-PMH could be directly provided by `ocrd`.

**Comment by mikegerber** [comment]
Yes, definitely an important task that I needed myself in the past!

**Comment by stweil** [comment]
I don't understand this issue. https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-497166623 is the direct link to the METS.

**Comment by kba** [comment]
> I don't understand this issue. https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-497166623 is the direct link to the METS.

It's not the METS itself, it's a [OAI-PMH GetRecord response](https://www.openarchives.org/OAI/openarchivesprotocol.html#GetRecord).

**Comment by mikegerber** [comment]
> > I don't understand this issue. [https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oaideslub-dresden:db:id-497166623](https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-497166623) is the direct link to the METS.
> 
> It's not the METS itself, it's a [OAI-PMH GetRecord response](https://www.openarchives.org/OAI/openarchivesprotocol.html#GetRecord).

Which contains the METS ;) 

@wrznr and me want an easy way to create a workspace from such a OAI-PMH response, i.e. make unwrapping the METS work with `clone`.

**Comment by bertsky** [comment]
I assume this was fixed exhaustively by #581?

**Comment by mikegerber** [comment]
I think this can be closed, as it's working fine:

~~~sh
% cd `mktemp -d`    
% ocrd workspace clone https://oai.sbb.berlin/\?verb\=GetRecord\&metadataPrefix\=mets\&identifier\=oai%3Adigital.staatsbibliothek-berlin.de%3APPN867445300
11:36:10.592 INFO ocrd_models.utils.is_oai_content - response data root.tag: '{http://www.openarchives.org/OAI/2.0/}OAI-PMH'
11:36:10.622 INFO ocrd.workspace.save_mets - Saving mets '/tmp/tmp.wkQDaCfgbP/mets.xml'
/tmp/tmp.wkQDaCfgbP
~~~
